start_bundle_coupled.R

```diff
@@ -152,9 +158,8 @@ message("run_compareScenarios:  ", run_compareScenarios)
 if (! file.exists("output")) dir.create("output")
 
 # Check if dependencies for a REMIND model run are fulfilled
-# Use ensureRequirementsInstalled(rerunPrompt="start_bundle_coupled.R") when coupled runs are using renv.
```

Comment:
I would maybe make that conditional on `is.null(renv::project())`. Or are snapshots discouraged in the sense "don't use that at all"? Then, line 8 should not be a warning, but a `stop`